src/Donations/index.tsx

```diff
@@ -170,6 +187,9 @@ function DonationInfo() {
           <BackButton color={"#000"} />
         </button>
       )}
+      {isApproved && isTopProject && router.query.to && (
+        <div className={"topProjectBadge theme-light"}>Top Project</div>
+      )}
```

Comment:
If you group the back button and top project badge into a single div, you could use flexbox and avoid absolute positioning for the top project badge.